tests/Builder/SourceFileRepositoryTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Website\Tests\Builder;
+
+use Doctrine\Website\Builder\SourceFile;
+use Doctrine\Website\Builder\SourceFileRepository;
+use PHPUnit\Framework\TestCase;
+use function usort;
+
+class SourceFileRepositoryTest extends TestCase
+{
+    /** @var string */
+    private $rootDir;
+
+    /** @var SourceFileRepository */
+    private $sourceFileRepository;
+
+    public function testGetFiles() : void
+    {
+        $buildDir = '';
+        $inPath   = 'source/blog';
+
+        $files = $this->sourceFileRepository->getFiles(
```

Comment:
If this test has more than one file in the return values, it is not strict enough by testing only one of the results. We should use fixtures here to control the amount of files that are returned.